### DISCUSSION TITLE
ESP WROOM 32 and Ydlidar x4

### DISCUSSION THREAD

**Discussion by g02200jeff** [makerspet/support#5] [Support Q&A] [answered]
My configuration is :
- ESP WROOM 32
-YDLIDAR x4
- L298N
- GA25 -370 

With your assistance (by messenger), the motors works and I can use Teleop. But now, my issue is about the YDLIDAR x4. The lidar start and stop every 2s, have you an idea about this ?

**Comment by makerspet** [comment]
Could you please post Arduino log output?

**Reply by g02200jeff** [reply, marked as answer]
I redid all the wire connections and now it works. The issue is resolved. Now I can move the robot with goals.
